etc/nyx.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for nyx
+# This file is overwritten after every install/update
+# Persistent local customizations
+include nyx.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${PATH}/python3*
+noblacklist /usr/include/python3*
+noblacklist /usr/lib/python3*
+noblacklist /usr/local/lib/python3*
+noblacklist /usr/share/python3*
+
+noblacklist ${HOME}/.nyx
+
+mkdir ${HOME}/.nyx
+
+whitelist ${HOME}/.nyx
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+# apparmor
+caps.drop all
+# ipc-namespace
+netfilter
+no3d
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+# tracelog
+
+disable-mnt
+private-bin nyx,python
+private-cache
+private-dev
+private-etc passwd,tor
```

Comment:
Maybe add `fonts`? Also, does nyx need internet connection?

Reply:
nyx should basically be a copy of arm.profile, a lot changed but the profile still works iirc

Reply:
arm profile is giving access to more binaries and etc, are they needed for nyx? it seems to work fine without

Reply:
@Fred-Barclay without protocol unix,inet,inet6 it's not possible to connect to the localhost tor, should we block internet using a netfilter rule?

Reply:
> arm profile is giving access to more binaries and etc

ah, it does not, I forgot nyx lost the ability to start Tor itself, and can only monitor now


for the protocol, most systems default to the control socket, so you can `protocol unix` and add a comment

Reply:
I'm mostly surprised this works without adding the likes of `resolv.conf` and whatnot to private-etc -- see, for example, the spotify and firefox-common profiles.

Reply:
@Fred-Barclay nyx only connects to Tor's control socket or port, it itself makes no connections to the Internet